packages/starlight/utils/collection.ts

```diff
@@ -1,8 +1,8 @@
 import { resolve } from 'node:path';
 import { fileURLToPath } from 'node:url';
 
-const collectionNames = ['docs', 'i18n'] as const;
-export type StarlightCollection = (typeof collectionNames)[number];
+const _collectionNames = ['docs', 'i18n'] as const;
+export type StarlightCollection = (typeof _collectionNames)[number];
```

Comment:
Is there a reason not to convert this just to a type if we’re not using the tuple anywhere? (I guess maybe we used to use the tuple in the past and this is a left over artifact of that.)

```suggestion
export type StarlightCollection = 'docs' | 'i18n';
```